src/component/skjema-visning/skjema-visning.tsx

```diff
@@ -78,12 +78,9 @@ export function SkjemaVisning(props: { fattetVedtak: Vedtak }) {
 				))}
 			</ul>
 
-			<button
-				className="skjema-visning__oyblikksbilde-lenke"
-				onClick={() => changeView(ViewType.OYBLIKKSBILDE_VISNING, { vedtakId: id })}
-			>
+			<Button variant="tertiary" onClick={() => changeView(ViewType.OYBLIKKSBILDE_VISNING, { vedtakId: id })}>
```

Comment:
Før var jo denne knappen kodet til å se ut som en lenke så jeg antar at det er et bevisst valg at vi har gått for en faktisk knapp (og ikke en lenke-lenk), om det gav mening? 😅

Reply:
Ja, bevisst forenkling!